### PR TITLE
Added login and signup pages

### PR DESCRIPTION
**Pull request by dbm19**

### CODE REVIEW

**Comment by hicks8989** [comment]
The code looks really good and the UI looks great! I just one issue when trying to run the code initially, the package.json file had an extra comma which crashed the program on "npm start", but outside of that it looks really good to me!

File: client/package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^4.1.3",
+    "flag-icon-css": "^3.3.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
```
Comment (hicks8989) [inline client/package.json]: The comma after "react-scripts" seems to be crashing the program on "npm start" for me.

**Review by hicks8989 — CHANGES_REQUESTED** [review]
The code looks really good and the UI looks great! I just one issue when trying to run the code initially, the package.json file had an extra comma which crashed the program on "npm start", but outside of that it looks really good to me!

File: client/package.json (hunk 1 above)
Comment (dbm19) [reply client/package.json]: Thanks Eric! I'll fix that comma asap.

File: client/src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            <p>Converse with anyone with any language.</p>
+          </div>
+        </div>
+      </Grid>
+      <Grid item xs={6} sm={8} md={5}>
+        <div className={classes.login}>
+          <div>
+            <p>Don't have an account?</p>
+          </div>
+          <Button
```
Comment (aeciorc) [inline client/src/pages/Login.js:86]: I noticed you didn't implement this yet, you can do that pretty easily with a Link component from react-router-dom and style it as a button

**Review by aeciorc — COMMENTED** [review]
This looks great, except for the font on the textfields, it is different from everywhere else

**Review by hicks8989 — APPROVED** [review]
Looks good!

**Pull request merged**